daemon/daemon_main.go

```diff
@@ -328,9 +298,8 @@ func checkMinRequirements() {
 	if err := os.Chdir(option.Config.LibDir); err != nil {
 		log.WithError(err).WithField(logfields.Path, option.Config.LibDir).Fatal("Could not change to runtime directory")
 	}
-	probeScript := filepath.Join(option.Config.BpfDir, "run_probes.sh")
-	if err := exec.Command(probeScript, option.Config.BpfDir, option.Config.StateDir).Run(); err != nil {
-		log.WithError(err).Fatal("BPF Verifier: NOT OK. Unable to run checker for bpf_features")
+	if err := probes.RunProbes(option.Config.BpfDir, option.Config.StateDir); err != nil {
+		log.WithError(err).Warn("BPF Verifier: NOT OK")
```

Comment:
This is different from what we currently do. Currently we `Fatal()` at this layer,  because for instance if the header file is only half-written then it will likely mess up all compilations from this point forward.

I suspect what you're trying to achieve here is to ignore the failure to fetch kernel config. If that's the case, then I think we should handle/ignore this further down in the kernel probe config, like we do in the current script.